operator/src/main/resources/Operator.properties

```diff
@@ -167,9 +167,11 @@ WLSDO-0019=DomainUID ''{0}'' and admin server name ''{1}'' combination ''{2}'' e
 WLSDO-0020=Online WebLogic configuration updates complete \
   but there are pending non-dynamic changes that require \
   pod restarts to take effect. The changes are:
-
+WLSDO-0021=DomainUID ''{0}'' server ''{1}'' does not have a port available for the operator to send REST calls. \
+  The default listen port and SSL port are disabled, the admin port is not configured and there is not a channel with the admin privileges.\
```

Comment:
Phrasing is clumsy. You could check with Rosemary, but I would recommend, "... and there is no channel with admin privileges."